src/style.css

```
@@ -0,0 +1,76 @@
+html {
+  font-family: "Arial", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: inline-flex;
```

Why inline flex? Common flex will do it
```suggestion
  display: flex;
```